src/CADL.Extension/Emitter.Csharp/src/lib/model.ts

```diff
@@ -426,6 +427,22 @@ export function getInputType(
 
             models.set(name, model);
 
+            if(discriminator) {
+                const discriminatorProp = {
+                    Name: discriminator.propertyName,
+                    SerializedName: discriminator.propertyName,
+                    Description: "",
```

Comment:
maybe we should set the `Description` like `discriminator property for inheritance`?

Reply:
@archerzz  No. in cadl `discriminator` is not defined as an property of base model, it just a decorator. There is no doc for it. So we cannot set Description for it.